.github/workflows/notebook_ci.yaml

```diff
@@ -61,11 +92,16 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/download-artifact@master
         with:
-          name: executed-notebooks
-          path: notebooks
+          path: publish
```

Comment:
If you don't pass any `name` to the action it downloads all artifacts (see [here](https://github.com/marketplace/actions/download-a-build-artifact#download-all-artifacts)).

Reply:
Ah, I see. Perhaps we should just explicitly download the two artifacts separately? If more artifacts are added later, it might be awkward to accidentally publish them to the web.